code/datums/gamemode/role/catbeast.dm

```diff
@@ -93,19 +124,38 @@ var/list/catbeast_names = list("Meowth","Fluffy","Subject 246","Experiment 35a",
 #define SURVIVAL_THREAT 1
 #define DEFILE_THREAT 0.75
 
+#define AREAS_THRESHOLD_TIER2 20
+#define TICKS_THRESHOLD_TIER2 (5 MINUTES)/20
```

Comment:
why not just write 15 SECONDS? unless my math is off...?

Reply:
Does it really work out to be 15 seconds? My intention was to make this 5 MINUTES, that's why it reads 5 MINUTES. I'll look this over in a bit.